src-docs/src/views/tool_tip/tool_tip.js

```diff
@@ -60,7 +60,7 @@ export default () => (
       <p>
         This tooltip appears on the bottom of this icon:{' '}
         <OuiToolTip position="bottom" content="Here is some tooltip text">
-          <OuiIcon tabIndex="0" type="alert" title="Icon with tooltip" />
```

Comment:
it seems really strange that the browser and the OUI Tooltip would appear at the same time. Can you provide clarification as to what you are referencing to validate its proper behavior?

Reply:
In the original issue, on of the items says `Confirm if on mouse over the OuiTooltip and the browser tooltip should be appearing at the same time in this example (seems like a bug)`. The behavior observed isn't a bug. The way it's written now, it's written to have both, but it indeed make more sense not to have the browser title. Just wanted to confirm

Reply:
When you say proper behavior, is there a spec somewhere? or what is the reason why both of those should show concurrently? Is it an accessibility issue, or other?

Reply:
From what I can see, this might just be a case where the icon was copy-pasted from somewhere else. In the `IconTip` section, the built-in browser tooltip isn't used at all, so it seems that it might have been unintentional here

Reply:
Lets go ahead and remove it from the docs site, as you mentioned it could have been an accident. I will take an action item separate for this to find examples where this might be on purpose in other design systems... 
Thanks @BSFishy  for diving in on this!